examples/github-stars-example/github-stars-example-job/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pandas
+plotly
+pygithub
+streamlit
+vdk-core
```

Comment:
vdk-core is not necessary here. AND is not advisable to add it as vdk version is managed by the platform when deployed